pyqtgraph/widgets/ColorMapWidget.py

```diff
@@ -71,7 +71,15 @@ def addNew(self, name):
         defaults = fieldSpec.get('defaults', {})
         for k, v in defaults.items():
             if k == 'colormap':
-                item.setValue(v)
+                if mode == 'range':
+                    item.setValue(v)
+                if mode == 'enum':
```

Comment:
let's change this second `if` to be `elif`

Reply:
Done!